karapace/compatibility/__init__.py

```diff
@@ -127,6 +132,19 @@ def check_compatibility(
                 )
             )
 
+    elif old_schema.schema_type is SchemaType.PROTOBUF:
+        if compatibility_mode in {CompatibilityModes.BACKWARD, CompatibilityModes.BACKWARD_TRANSITIVE}:
+            result = check_protobuf_compatibility(
+                reader=new_schema.schema,
+                writer=old_schema.schema,
+            )
+        elif compatibility_mode in {CompatibilityModes.FORWARD, CompatibilityModes.FORWARD_TRANSITIVE}:
+            result = check_protobuf_compatibility(reader=old_schema.schema, writer=new_schema.schema)
```

Comment:
> Just a small detail, having a comma after the last argument will make the formatting consistent.

Is automatic code formatter used in the project? It is hard to keep track on this manually while linter allows it.

Reply:
Yeah, the code is formatted with [YAPF](https://github.com/google/yapf). [This is how](https://github.com/aiven/karapace/blob/be7c748449f5a696c421bddd698ee78adb77797c/Makefile#L83-L92) you can use it. I only pointed this out because the tool will not add the comma for you, it will only format if it is there. It is a small detail really, I just find it easier to read :)

Reply:
Thanks for pointing out this. @libretto, please note.